clusters/build-clusters/common_except_app.ci/loki.yaml

```diff
@@ -253,6 +253,8 @@ spec:
         name: shared-data
   updateStrategy:
     type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 50%
```

Comment:
Perhaps `33%` (as we have 3 masters) would work better?

Reply:
Updated